tests/test_adjoint_jacobian.py

```diff
@@ -800,8 +806,8 @@ def f(params1, params2):
             qml.RY(jax.numpy.cos(params2), wires=[0])
             return qml.expval(qml.PauliZ(0))
 
-        params1 = jax.numpy.array(0.3)
-        params2 = jax.numpy.array(0.4)
+        params1 = jax.numpy.array(0.3, dev.R_DTYPE)
+        params2 = jax.numpy.array(0.4, dev.R_DTYPE)
```

Comment:
I think this is not required after introducing `config.update("jax_enable_x64", True)`.

Reply:
Since it doesn't hurt, I'd keep it for now. We can strip this kinda thing easily later